Xamarin.Forms.Core/Button.cs

```diff
@@ -143,33 +136,33 @@ public Color TextColor
 			set { SetValue(TextElement.TextColorProperty, value); }
 		}
 
-		bool IButtonController.IsEnabledCore
+		bool IButtonElement.IsEnabledCore
 		{
 			set { SetValueCore(IsEnabledProperty, value); }
 		}
 
 		[EditorBrowsable(EditorBrowsableState.Never)]
-		public void SendClicked() => ButtonElementManager.ElementClicked(this, this);
+		public void SendClicked() => ButtonElement.ElementClicked(this, this);
```

Comment:
it doesn't feel right to have this in the `ButtonElement`

Reply:
Because it has to do with an *ElementClick* ? 
So it seems like ButtonElement is too specific? It should maybe be part of *CommandElement* or  *ClickableElement* ?

Or is this more about a controller vs element thing?